charts/flink-job/templates/flinkjobcluster.yaml

```diff
@@ -11,70 +11,72 @@ spec:
   image:
     name: {{ template "flink-job-cluster.imageFqdn" . }}
     pullPolicy: IfNotPresent
+  batchSchedulerName: {{ .Values.batchSchedulerName }}
   envVars:
 {{ toYaml .Values.envVars | indent 4 }}
   serviceAccountName: {{ template "flink-job-cluster.fullname" . }}
   jobManager:
     accessScope: {{ .Values.jobManager.accessScope }}
     ports:
       ui: {{ .Values.jobManager.ports.ui }}
+      # TODO (Aaron/Albert) expose the other ports?
+{{- if .Values.jobManager.metrics.enabled }}
+    extraPorts:
```

Comment:
I rearranged some stuff here to make it align with the flink-on-k8s-operator docs better